src/main/webapp/WEB-INF/tags/teammatesLogo.tag

```diff
@@ -0,0 +1,5 @@
+<%@ tag description="TEAMMATES logo brand"%>
+
+<a class="navbar-brand" href="/">
+    <img height="100%" src="../images/teammateslogo.jpg" alt="TEAMMATES[Logo] - Online Peer Feedback/Evaluation System for Student Team Projects">
```

Comment:
Could this be changed to the [root-relative URL](https://github.com/TEAMMATES/teammates/pull/6836#discussion_r106812391) mentioned above?